### PR TITLE
Favorites view name can be random

### PR DESCRIPTION
**Pull request by 163gal**
GCI Task: https://codein.withgoogle.com/tasks/4788542454628352/
Now the favorite views can have names (Or if they doesn't have one, they receive one).

### CODE REVIEW

**Comment by quozl** [comment]
Thanks for your contribution.  I've reviewed the patch.

Why are there two assignments to variable favorite_icons, one of which is a getter of get_view_labels?

I don't understand how to test the patch.  How can a favourite view have a name?

The patch description should say what the problem is, how it is fixed, and how it can be tested.

I suggest you make changes and do a force push for further review.  I'm not inclined to review a different pull request, as I'll lose my comments.

**Comment by i5o** [comment]
Hey @quozl 

I will fix the description and those things, its for a gci task.
This is for the feature "multiple homeviews", before gsettings port it was possible to change the homeviews names (using  the activity "Manage homeviews": https://github.com/i5o/manage-homeviews )
Looks like it was lost after the gsettings port.

File: src/jarabe/model/desktop.py [hunk 1]
```diff
@@ -58,6 +61,11 @@ def get_favorite_icons(self):
 
     favorite_icons = GObject.property(type=object, getter=get_favorite_icons)
 
+    def get_view_labels(self):
+        return self._view_labels
+
+    favorite_icons = GObject.property(type=object, getter=get_view_labels)
```
Comment (samdroid-apps) [inline src/jarabe/model/desktop.py]: Typo here

**Comment by samdroid-apps** [comment]
Please make a patch for manage home views so that i can test.

**Comment by i5o** [comment]
@samdroid-apps I recommend you to test with https://wiki.sugarlabs.org/go/Features/Multiple_home_views gsettings instructions for Sugar 102+
adding view-label key..

That is another GCI task, and I think student aren't able to work on two at the same time.

**Comment by quozl** [comment]
@i5o in IRC referenced https://wiki.sugarlabs.org/go/Features/Multiple_home_views which in my opinion may need editing which may form part of the overall work, and would be a great thing to mention in the patch description.

That this pull request is associated with a GCI task, or that coordinating multiple students is required, shouldn't be part of the pull request discussion.

**Comment by 163gal** [comment]
I'm fixing the typo.

**Pull request closed without merging**